mava/systems/rec_ippo_rware.py

```diff
@@ -48,10 +48,11 @@
     RecActorApply,
     RecCriticApply,
     RNNLearnerState,
+    RnnObservation,
```

Comment:
just for consistency can you name it RNNObservation or RecObservation 😅

Reply:
I think RecObservation may be better if you agree on this I'll change it 👌